scipy/stats/_hypotests.py

```diff
@@ -3,11 +3,10 @@
 import numpy as np
 import warnings
 from itertools import combinations
-import scipy.stats
+from scipy import stats
+from scipy.stats import _stats_py
 from scipy.optimize import shgo
-from . import distributions
```

Comment:
Again, part of an attempt to resolve circular imports, but that's no longer a problem, so these can be removed if we prefer.